en/assembly/quick_start_cuav_v5_plus.md

```diff
@@ -74,12 +74,10 @@ If using a Spektrum satellite receiver, this should be connected to the flight c
 
 ## Power
 
-The V5+ kit includes the *HV\_PM* module, which supports 2~10S LiPo batteries.
+The V5+ kit includes the *HV\_PM* module, which supports 2~10S LiPo batteries. 
```

Comment:
@cuhome In the nano, you changed this to 14s max. Should it be changed here too?

Reply:
@cuhome see ^^^^

Reply:
yes, the correct data is 2 ~ 14S.

Reply:
Fixed